apps/judicial-system/web/src/components/VerdictStatusAlert/VerdictStatusAlert.tsx

```diff
@@ -105,6 +105,18 @@ const VerdictStatusAlertMessage = ({
 
   const isServed = Boolean(verdict?.serviceDate && verdict?.serviceStatus)
   if (isServed) {
+    if (verdict.serviceStatus === VerdictServiceStatus.LEGAL_PAPER) {
+      return (
+        <AlertMessage
+          type="info"
+          title={`Dómur sendur í Lögbirtingarblaðið - ${defendantName}`}
```

Comment:
Here we should prob say: "Dómur birtur í Lögbirtingarblaðinu" and then we should remove the timestamp, since there is no timestamp on the publish date for Lögbirtingur.

Reply:
I believe we will have a timestamp for when the actual birting took place. I think ríksak will manually (at least to begin with) enter a date when the paper is published.